setup.py

```diff
@@ -19,12 +19,12 @@
         'pyyaml == 5.3.1',
         'pandas == 1.0.3',
         'pytest == 5.4.1',
-        'requests - mock == 1.7.0',
+        'requests-mock == 1.7.0',
```

Comment:
Do not put pytest and requests-mock dependencies here - they are used only for testing. User does not need them. Put them in requirements.txt. Also, in setup.py, do not restrict to specific version of django - you use tox, to make sure that it will work with various versions.